network/codec/cbor/envelope.go

```diff
@@ -49,5 +49,8 @@ const (
 	// testing
 	CodeEcho
 
+	// DKG
+	CodeDKGMessage
+
```

Comment:
@yhassanzadeh13 thanks for the comment!

FYI I believe `CodeDKGMessage` got merged yesterday into master [1] by Jordan! I merged master into this PR branch later yesterday and that caused integration issues for me. Why? Because there is a new function which looks up the human readable name for the code and it didn't exist yet for master or this branch. But I fixed that yesterday evening.

[1] https://github.com/onflow/flow-go/blob/master/network/codec/json/envelope.go#L56